### PR TITLE
Feature/confirm job

### PR DESCRIPTION
**Pull request by pengin7384**
Done
- Add 'Confirm' job

### CODE REVIEW

File: worker_node/job/confirm.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const Ain = require('@ainblockchain/ain-js').default;
+
+class Confirm extends Base {
+  static configProps = [
+    'ainUrl',
+    'startBlockNumber',
+    'finishBlockNumber',
+    'txHashList',
+  ];
+  #ain;
```
Comment (platfowner) [inline worker_node/job/confirm.js:11]: #ain = null; ?

**Review by platfowner — APPROVED** [review]
LGTM
with some minor comments.

File: worker_node/job/confirm.js (hunk 1 above)
Comment (pengin7384) [reply worker_node/job/confirm.js:11]: I used `#ain` as the meaning of declaration and initialized it in `constructor`. ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)) If I initialize it to `null`, the IDE infers the type to `null`. As a result it gives warnings like this 😢 
image

**Comment by pengin7384** [comment]
Thanks for the review! Merged!

**Pull request merged**